### PR TITLE
Get Activity

### PR DESCRIPTION
**Pull request by woomurf**
- add event.getActivity
- add event.updateActivityStatus

### CODE REVIEW

File: src/event.ts [hunk 1]
```diff
@@ -168,6 +169,76 @@ export default class Event extends AinftBase {
...
+    return axios
+      .put(`${this.baseUrl}/${eventId}/activity/${activityId}`, body, {
+        headers: {
+          'X-AINFT-Date': timestamp,
+          Authorization: `AINFT ${signature}`,
+        },
+      })
+      .then((res) => res.data)
+      .catch((e) => {
+        throw e.response?.data;
```
Comment (liayoo) [inline src/event.ts:238]: do we need the `?` for other errors as well?

**Review by liayoo — APPROVED** [review]
LGTM with a comment, thanks!!

File: src/event.ts (hunk 1 above)
Comment (woomurf) [reply src/event.ts:238]: Maybe Yes...  
I forgot to start the server in local and experienced an error in that part while testing.
Even if the server dies, it seems necessary for the SDK to handle errors well.
Can I fix it in this PR? Or the next PR?

Comment (liayoo) [reply src/event.ts:238]: I see! Let's fix it in the next PR (and possibly include it in the next next release, depending on the ETA of @ehgmsdk20 ?)

Comment (woomurf) [reply src/event.ts:238]: Thanks for review~!

**Pull request merged**